src/AgoraGovernor.sol

```diff
@@ -15,6 +15,7 @@ import {GovernorSettingsUpgradeableV2} from "src/lib/openzeppelin/v2/GovernorSet
 import {GovernorTimelockControlUpgradeableV2} from "src/lib/openzeppelin/v2/GovernorTimelockControlUpgradeableV2.sol";
 import {IProposalTypesConfigurator} from "src/interfaces/IProposalTypesConfigurator.sol";
 import {VotingModule} from "src/modules/VotingModule.sol";
+import {ScopeKey} from "src/ScopeKey.sol";
```

Comment:
i think we should not have this lib for now and instead have it as an internal function here and in tests

Reply:
i would also not use the `Packing` lib from OZ yet as it hasnt been audited. we can grab `pack_20_4` and modify it for this need and remove the OZ lib. once OZ releases, we can update the contracts-v5 lib and replace with the OZ function.